types/webpack/index.d.ts

```diff
@@ -1196,6 +1196,8 @@ declare namespace webpack {
         interface MultiStats {
             stats: Stats[];
             hash: string;
+            /** Returns true if there were errors while compiling. */
```

Comment:
```suggestion
            /** Returns true if a child compilation encountered an error. */
```
I think it would be better to use the same comment here as in the official annotation: https://github.com/webpack/webpack/blob/0300a1f7dc6cc4b9f4a5848ef38f965f2966ab54/lib/MultiStats.js#L27

(The annotation itself comes from the v5 but it does still the same thing like in v4 https://github.com/webpack/webpack/blob/11e94dd2d0a8d8baae75e715ff8a69f27a9e3014/lib/MultiStats.js#L18)